libavcodec/x86/vvc_itx_1d.asm

```diff
@@ -0,0 +1,1142 @@
+                               m12_1, m12_2, m12_3, m12_4, m12_5, m12_6, m12_7, m12_8, m12_9, m12_10, m12_11, m12_12, m12_13, m12_14, m12_15, m12_16, \
+                               m13_1, m13_2, m13_3, m13_4, m13_5, m13_6, m13_7, m13_8, m13_9, m13_10, m13_11, m13_12, m13_13, m13_14, m13_15, m13_16, \
+                               m14_1, m14_2, m14_3, m14_4, m14_5, m14_6, m14_7, m14_8, m14_9, m14_10, m14_11, m14_12, m14_13, m14_14, m14_15, m14_16, \
+                               m15_1, m15_2, m15_3, m15_4, m15_5, m15_6, m15_7, m15_8, m15_9, m15_10, m15_11, m15_12, m15_13, m15_14, m15_15, m15_16, \
+                               m16_1, m16_2, m16_3, m16_4, m16_5, m16_6, m16_7, m16_8, m16_9, m16_10, m16_11, m16_12, m16_13, m16_14, m16_15, m16_16) \
+     m1_1,  m1_2,  m1_3,  m1_4,  m1_5,  m1_6,  m1_7,  m1_8,  m1_9,  m1_10,  m1_11,  m1_12,  m1_13,  m1_14,  m1_15,  m1_16, \
+     m2_1,  m2_2,  m2_3,  m2_4,  m2_5,  m2_6,  m2_7,  m2_8,  m2_9,  m2_10,  m2_11,  m2_12,  m2_13,  m2_14,  m2_15,  m2_16, \
+     m3_1,  m3_2,  m3_3,  m3_4,  m3_5,  m3_6,  m3_7,  m3_8,  m3_9,  m3_10,  m3_11,  m3_12,  m3_13,  m3_14,  m3_15,  m3_16, \
+     m4_1,  m4_2,  m4_3,  m4_4,  m4_5,  m4_6,  m4_7,  m4_8,  m4_9,  m4_10,  m4_11,  m4_12,  m4_13,  m4_14,  m4_15,  m4_16, \
+     m8_1,  m8_2,  m8_3,  m8_4,  m8_5,  m8_6,  m8_7,  m8_8,  m8_9,  m8_10,  m8_11,  m8_12,  m8_13,  m8_14,  m8_15,  m8_16, \
+     m7_1,  m7_2,  m7_3,  m7_4,  m7_5,  m7_6,  m7_7,  m7_8,  m7_9,  m7_10,  m7_11,  m7_12,  m7_13,  m7_14,  m7_15,  m7_16, \
+     m6_1,  m6_2,  m6_3,  m6_4,  m6_5,  m6_6,  m6_7,  m6_8,  m6_9,  m6_10,  m6_11,  m6_12,  m6_13,  m6_14,  m6_15,  m6_16, \
+     m5_1,  m5_2,  m5_3,  m5_4,  m5_5,  m5_6,  m5_7,  m5_8,  m5_9,  m5_10,  m5_11,  m5_12,  m5_13,  m5_14,  m5_15,  m5_16, \
+     m9_1,  m9_2,  m9_3,  m9_4,  m9_5,  m9_6,  m9_7,  m9_8,  m9_9,  m9_10,  m9_11,  m9_12,  m9_13,  m9_14,  m9_15,  m9_16, \
+    m10_1, m10_2, m10_3, m10_4, m10_5, m10_6, m10_7, m10_8, m10_9, m10_10, m10_11, m10_12, m10_13, m10_14, m10_15, m10_16, \
+    m11_1, m11_2, m11_3, m11_4, m11_5, m11_6, m11_7, m11_8, m11_9, m11_10, m11_11, m11_12, m11_13, m11_14, m11_15, m11_16, \
+    m12_1, m12_2, m12_3, m12_4, m12_5, m12_6, m12_7, m12_8, m12_9, m12_10, m12_11, m12_12, m12_13, m12_14, m12_15, m12_16, \
+    m16_1, m16_2, m16_3, m16_4, m16_5, m16_6, m16_7, m16_8, m16_9, m16_10, m16_11, m16_12, m16_13, m16_14, m16_15, m16_16, \
+    m15_1, m15_2, m15_3, m15_4, m15_5, m15_6, m15_7, m15_8, m15_9, m15_10, m15_11, m15_12, m15_13, m15_14, m15_15, m15_16, \
+    m14_1, m14_2, m14_3, m14_4, m14_5, m14_6, m14_7, m14_8, m14_9, m14_10, m14_11, m14_12, m14_13, m14_14, m14_15, m14_16, \
+    m13_1, m13_2, m13_3, m13_4, m13_5, m13_6, m13_7, m13_8, m13_9, m13_10, m13_11, m13_12, m13_13, m13_14, m13_15, m13_16
+
+const vvc_dct2_32_odd_mat, dw matvec_mul_16_permute(dct2_32_odd_mat_permute( \
```

Comment:
vvc_itx_1d.asm:181: warning: single-line macro `matvec_mul_16_permute' exists, but not taking 1 parameter [-w+pp-macro-params-single]

Reply:
See https://github.com/ffvvc/FFmpeg/pull/114#discussion_r1269083847